app/src/main/java/com/muzima/adapters/observations/ObservationsByEncounterBackgroundTask.java

```diff
@@ -87,7 +128,7 @@ public int compare(Encounter lhs, Encounter rhs) {
     @Override
     protected void onPostExecute(Encounters encountersWithObservations) {
         if (encountersWithObservations == null) {
-            Toast.makeText(observationsByEncounterAdapter.getContext(), observationsByEncounterAdapter.getContext().getString(R.string.error_observation_fetch), Toast.LENGTH_SHORT).show();
+            Toast.makeText(observationsByEncounterAdapter.getContext(),"This patient does not have any SHR data.", Toast.LENGTH_SHORT).show();
```

Comment:
Shouldn't  we have two Toast messages, one when using SHR and another when not using SHR using if condition as ones in doInBackground